src/cms/templates/linkcheck/links_by_filter.html

```diff
@@ -0,0 +1,59 @@
+{% extends '_base.html' %}
+{% load i18n %}
+
+{% block content %}
+  <h1 class="heading">
+    {% if link_filter == 'valid' %}
+      {% trans 'Valid links' %} ({{ number_valid }})
+    {% elif link_filter == 'invalid' %}
+      {% trans 'Invalid links' %} ({{ number_invalid }})
+    {% elif link_filter == 'unchecked' %}
+      {% trans 'Unchecked links' %} ({{ number_unchecked }})
+    {% else %}
+      {% trans 'Ignored links' %} ({{ number_ignored }})
+    {% endif %}
+  </h1>
+  <a href="{% if link_filter == 'valid' %}#{% else %}{% url 'linkcheck' region_slug=region_slug link_filter='valid' %}{% endif %}" 
+    class="pr-2 {% if link_filter == 'valid' %}text-black font-bold cursor-default{% else %}bg-transparent hover:underline text-blue-500 hover:text-blue-600 cursor-pointer{% endif %}">
+    {% trans 'Valid' %}({{ number_valid }})
```

Comment:
```suggestion
    {% trans 'Valid' %} <span class="{% if link_filter == 'valid' %}text-gray-500 font-normal{% endif %}">({{ number_valid }})</span>
```
I propose a slightly different styling of the counters...